app/models/publisher_date_fields.rb

```diff
@@ -0,0 +1,17 @@
+module PublisherDateFields
+  DATE_FIELDS = {
+    publisher_first_published_at: :first_published_at,
+    publisher_last_edited_at: :last_edited_at,
+    publisher_major_published_at: :major_published_at,
+    publisher_published_at: :minor_published_at,
```

Comment:
I think this would be just `published_at` still

Reply:
Do you mean `publisher_published_at` should be renamed `published_at`?  Or that the param being sent should be called `published_at`?

Reply:
Sorry: `publisher_published_at: :minor_published_at` => `publisher_published_at: :published_at`

Reply:
Cool 👍